shared/constants/waiting.tsx

```diff
@@ -5,6 +5,11 @@ export const anyWaiting = (state: Container.TypedState, ...keys: Array<string>)
   return !!keys.some(k => (state.waiting.counts.get(k) || 0) > 0)
 }
 
+export const useWaiting = (...keys: Array<string>) => {
```

Comment:
this seemed cleaner than the alternative. Open to discussion on how it works / is implemented.

Reply:
this is in `util/container` as `useAnyWaiting`